ComposeSample/README.md

```diff
@@ -0,0 +1,65 @@
+---
+page_type: sample
+name: "Surface Duo - ComposeSample"
```

Comment:
Joy since this repo is going to have now just Jetpack Compose samples and code, maybe is worth to change the name (folder and references) of this sample for one that is more specific? (since now all samples here will be based on Compose) so i's not confusing and we know just by reading its name what this sample is about.

Reply:
Good point. I am so bad at the naming, so just call it `Compose Gallery`, the first thing coming up to my mind.